### ISSUE TITLE
[BUG] Fails with invalid name when installing asdf-nodejs plugin

### ISSUE THREAD

**Issue by andrewcrook** [asdf-community/asdf-plugin-manager#33]
**Describe the bug**

When adding nodjs plugin it fails saying the name doesn’t flow rules regarding name case and the use of underline and hyphen. 
I cannot see anything wrong with the name compared to others.

**Steps to reproduce**

```
> cat  .plugin-versions
asdf-plugin-manager  https://github.com/asdf-community/asdf-plugin-manager.git  07a8a5a
direnv               https://github.com/asdf-community/asdf-direnv.git          2b649c8
lua                  https://github.com/Stratus3D/asdf-lua.git                  36fae6e
luajit               https://github.com/smashedtoatoms/asdf-luaJIT.git          bbfe3cd
nodejs               https://github.com/asdf-vm/asdf-nodejs.git                 9c4e0f2
python               https://github.com/danhper/asdf-python.git                 5e277e2
```

```
asdf-plugin-manager add nodejs
[INFO] Adding:
 is invalid. Name may only contain lowercase letters, numbers, '_', and ‘-'
```

**Expected behavior**

install the nodejs plugin 

**Screenshots**
**Additional context**

output has been posted above

**Comment by aabouzaid** [comment]
Hi @andrewcrook :wave: 
Which `asdf` and `asdf-plugin-manager` version do you have?

**Comment by aabouzaid** [comment]
The only case where that happens is when the `.plugin-versions` is not in the correct path (it should be a relative path when `asdf-plugin-manager` is executed).

Fixed in https://github.com/asdf-community/asdf-plugin-manager/pull/43 and available in [v1.3.0](https://github.com/asdf-community/asdf-plugin-manager/releases/tag/v1.3.0)
